docs/section6/Rule6-9.md

```diff
@@ -35,12 +37,16 @@
         - Get normalized space lighting schedule for B_RMR: `normalized_schedule_b = normalize_space_schedules(space_b.interior_lighting)`
 
       - Get normalized space lighting schedule for P_RMR: `normalized_schedule_p = normalize_space_schedules(space_p.interior_lighting)`
+      
+      - Check if lighting uses schedule to model occupancy control, get occupancy control credit: `if interior_lighting.are_schedules_used_for_modeling_occupancy_control: control_credit = data_lookup(table_G3_7, space.lighting_space_type, interior_lighting.occupancy_control_type)`
```

Comment:
For buildings <5000 SF credit for occupancy controls should be 10%. Table G3.7 shouldn't be used. (Control Credit=0.1)